dist/madrone-editoria11y.js

```diff
@@ -59,6 +62,34 @@ document.addEventListener('ed11yRunCustomTests', () => {
     `,
   }
 
+  Ed11y.M.missingHeadingLevel1 = {
+    title: 'Missing Heading Level 1',
+    tip: () => `
+<p>It seems you've not added a level 1 heading(h1) on your page. The Page Title typically serves as the level 1 heading.</p>
+<p>If you've chosen to hide the Page Title, please ensure to add the <em>Title</em> block in Layout Builder. To add the Title block to your page in Layout Builder:
+<ol>
+<li>Filter by block name <em>Title</em></li>
+<li>Set label to <em>- Hidden -</em></li>
+<li>Change the Formatter to <em>Text field formatter</em></li>
+<li>Set the Field wrapper to <em>H1</em></li>
+<li>Add/adjust your spacing as needed.</li>
+</ol>
+If the Page Title does not serve your needs for a level 1 heading, please ensure that you have a custom one created in the Text Editor.
+</p>
+<p>Remember, having one level 1 heading is important for good SEO practices, accessibility, and content organization.</p>
+    `
+  }
+
+  Ed11y.M.multipleHeadingLevel1 = {
+    title: 'Multiple Heading Level 1',
+    tip: () => `
+<p>You have <strong>multiple level 1 headings(h1)</strong> on your page. Typically, your page title is the level 1 heading of a page. You need to change the other level 1 headings to heading 2. You can learn more about heading structure for webpages on the <a href="https://drupal.oregonstate.edu/site-building-guide/elements">Site Building Guide</a> or <a href ="https://accessibility.oregonstate.edu/headings">the Accessibility website.</a></p>
+<p>This can cause confusion for your site visitors, particularly for those using assistive technologies like screen readers. The  level 1 heading is denote what your entire webpage is about.</p>
```

Comment:
The level 1 heading is >**to**< denote. This sentence also has an extra space after `The`

Reply:
Should be fixed now